toffy/panel_utils_test.py

```diff
@@ -0,0 +1,132 @@
+import os
+import csv
+import tempfile
+import pandas as pd
+import pytest
+
+from toffy import panel_utils
+from ark.utils import test_utils
+
+
+def test_drop_duplicate_masses():
+    duplicate_panel = pd.DataFrame({
+        'Mass': [1, 2, 3, 3, 1],
+        'Target': ['target1', 'target2', 'target3', 'target4', 'target5']
+    })
+
+    unique_panel = panel_utils.drop_duplicate_masses(duplicate_panel)
+
+    # check for no duplicate masses
+    assert len(list(unique_panel['Mass'])) == len(set(unique_panel['Mass']))
+
+    # check for correct edited target names
+    assert list(unique_panel['Target']) == ['target1_target5', 'target2', 'target3_target4']
```

Comment:
Add a case with 3 targets on same mass. 

In general, always good practice to check [1, 2, many] as the conditions for a function, where many can be three, 10, 100, etc